resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        <pre class="code--block">
+        {
+          "investorId": "1234567899",
+          "creationReason": "New",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_EXISTS",
+          "message": "This investor already has a LISA account",
+          "accountId": "1234567890"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: createOrTransferLisaAccount
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsRequest'
+            examples:
+              example-1:
+                value:
+                  investorId: '9876543210'
+                  creationReason: New
+                  accountId: '1234567890'
+                  firstSubscriptionDate: 2017-04-06
+              example-2:
+                value:
+                  investorId: '9876543210'
+                  creationReason: Transferred
+                  accountId: '1234567891'
+                  firstSubscriptionDate: 2017-04-06
+                  transferAccount:
+                    transferredFromAccountId: '8765432100'
+                    transferredFromLMRN: Z654321
+                    transferInDate: 2017-04-06
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsResponse'
+              examples:
+                example-1:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Account created
+                      accountId: '1234567890'
+                example-2:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Account transferred
+                      accountId: '1234567891'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                badRequest:
+                  $ref: '#/components/examples/invalidLmrn'
```

Comment:
This has been duplicated should be updated to: `$ref: '#/components/examples/badRequest'`